### ISSUE TITLE
Getopt::Type

### ISSUE THREAD

**Issue by AlexDaniel** [perl6/ecosystem-unbitrot#407]
Module [Getopt::Type](https://modules.perl6.org/dist/Getopt::Type) cannot be installed (`AlwaysFail`), perhaps it has some failing tests.

* Tickets are opened/closed in this repo automatically (though not immediately).
* If you can **install the module without any problems**, add `works for me` label, leave a comment saying that it works for you and mention any details that you feel are important.
* If it **needs a native library**, put `native dependency` label, describe what you did to install it and ensure that same instructions are present in the README file of the module (otherwise submit a pull request). Also try to update [this wiki page](https://github.com/perl6/ecosystem-unbitrot/wiki).
* If the **module is broken**, try to fix it and send a PR. Add `PR sent` label.
* If there is a **problem in one of the dependencies**, add `failing dependency` label and write a comment explaining the situation. Feel free to work on the corresponding ticket for the failing dependency.
* It is a good idea to **assign yourself** to this ticket if you're working on it (to make sure two or more people are not working on the same ticket at the same time).
* **Once you are done**, [search for a next ticket](https://github.com/perl6/ecosystem-unbitrot/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3A%22PR+sent%22+-label%3A%22deprecated+module%22+-label%3A%22native+dependency%22).

If you can't self-assign yourself or attach a label, please let us know on [#perl6 channel on freenode](https://perl6.org/irc) or just leave a comment here. We will try to give you privileges as fast as possible.

Output:

````````````````````````````````````````````````````````
===> Searching for: Getopt::Type
===> Found: Getopt::Type:ver<0.1> [via Zef::Repository::Ecosystems<p6c>]
===> Fetching [OK]: Getopt::Type:ver<0.1> to /home/alex/Blin/data/zef-data/tmp/1549580239.2961.6647/Getopt-Type.git
===> Extraction [OK]: Getopt::Type to /home/alex/Blin/data/zef-data/store/Getopt-Type.git
===> Testing: Getopt::Type:ver<0.1>
# Failed test at t/01-basic.t line 9
# Parameter '$opts' expected a writable container, but got Hash value
# Failed test at t/01-basic.t line 10
# Failed test at t/01-basic.t line 12
# Parameter '$opts' expected a writable container, but got Hash value
# Failed test at t/01-basic.t line 14
# Failed test at t/01-basic.t line 28
# Parameter '$opts' expected a writable container, but got Hash value
# Failed test at t/01-basic.t line 24
# Failed test at t/01-basic.t line 24
# Failed test at t/01-basic.t line 25
# Failed test at t/01-basic.t line 25
# Failed test at t/01-basic.t line 30
# Parameter '$opts' expected a writable container, but got Hash value
# Failed test at t/01-basic.t line 24
# Failed test at t/01-basic.t line 24
# Failed test at t/01-basic.t line 25
# Failed test at t/01-basic.t line 25
# Failed test at t/01-basic.t line 32
# Parameter '$opts' expected a writable container, but got Hash value
# Failed test at t/01-basic.t line 24
# Failed test at t/01-basic.t line 24
# Failed test at t/01-basic.t line 25
# Failed test at t/01-basic.t line 25
# Failed test at t/01-basic.t line 34
# Parameter '$opts' expected a writable container, but got Hash value
# Failed test at t/01-basic.t line 24
# Failed test at t/01-basic.t line 24
# Failed test at t/01-basic.t line 25
# Failed test at t/01-basic.t line 25
# Failed test at t/01-basic.t line 36
# Parameter '$opts' expected a writable container, but got Hash value
# Failed test at t/01-basic.t line 24
# Failed test at t/01-basic.t line 24
# Failed test at t/01-basic.t line 25
# Failed test at t/01-basic.t line 25
# Looks like you failed 29 tests of 42
t/01-basic.t .. 
Dubious, test returned 29 (wstat 7424, 0x1d00)
Failed 29/42 subtests 

Test Summary Report
-------------------
t/01-basic.t (Wstat: 7424 Tests: 42 Failed: 29)
  Failed tests:  1-2, 4, 6, 8-12, 15-19, 22-26, 29-33, 36-40
  Non-zero exit status: 29
Files=1, Tests=42,  2 wallclock secs ( 0.04 usr  0.00 sys +  1.60 cusr  0.13 csys =  1.77 CPU)
Result: FAIL
===> Testing [FAIL]: Getopt::Type:ver<0.1>
Failed to get passing tests, but continuing with --force-test
===> Installing: Getopt::Type:ver<0.1>
===> Install [OK] for Getopt::Type:ver<0.1>

````````````````````````````````````````````````````````

Ping @tadzik

**Comment by JJ** [comment]
There's [this PR](https://github.com/tadzik/Getopt-Type/pull/3) which has not been accepted. I think I'll opt for an auth release in this case too.

**Comment by JJ** [comment]
Accepted that PR and done a few other goodies, released [this as version 0.1.1](https://github.com/JJ/Getopt-Type) Will mark as PR sent *and* accepted.
